inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/registry/WorkloadRegistryImpl.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.workload.registry;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.inception.workload.extension.WorkloadExtension;
+import de.tudarmstadt.ukp.inception.workload.model.WorkloadManagementService;
+
+@Component
+public class WorkloadRegistryImpl
+    implements WorkloadRegistry
+{
+    private List<WorkloadExtension> extensions;
+    private @Autowired WorkloadManagementService workloadManagementService;
+
+    public WorkloadRegistryImpl(
+        @Lazy @Autowired(required = false) List<WorkloadExtension> aExtensions)
+    {
```

Comment:
Please make us of the `ExtensionPoint_ImplBase` base class for this registry unless there is a strong reason not to do it (and if so, please let me know the reason).